### PR TITLE
Update aspnetcore interop scripts to fix the interop build

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This might help unblock https://github.com/grpc/grpc-dotnet/pull/205.

This uses the updated method to add the installed dotnet SDK to the path.

cc @jtattermusch

### CODE REVIEW

File: templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template [hunk 1]
```diff
@@ -25,14 +25,11 @@
...
-  
+
   # If needed, update dotnet SDK and put it on path
   ./build/get-dotnet.sh
-  if [ -f $HOME/.dotnet/dotnet ]
-  then
-    ln -s $HOME/.dotnet/dotnet /usr/local/bin/dotnet
-  fi
-  
+  source ./activate.sh
```
Comment (JamesNK) [inline templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template]: There is a circular dependency here between this PR and https://github.com/grpc/grpc-dotnet/pull/205. activate.sh doesn't exist in the grpc-dotnet repo yet.

What is the best way to resolve this? Would merging activate.sh in a separate PR help?

Comment (JunTaoLuo) [reply templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template]: Activate.sh would only make sense if we made the change to install to a local .dotnet/ directory. Is it possible to merge while ignoring failing tests? I'm thinking we might need to merge the changes in grpc-dotnet without the interop tests passing and then merge this as a reaction PR.

Comment (JamesNK) [reply templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template]: We can't. An admin (@jtattermusch?) should be able to.

**Comment by jtattermusch** [comment]
Sorry, haven't seen this PR so I just created https://github.com/grpc/grpc/pull/18740/files

**Comment by jtattermusch** [comment]
Oh and it turns out this fix is not enough, because source ./activate.sh adds dotnet to path for the build, but not when the interop server is being executed later. I'm becoming less excited about https://github.com/grpc/grpc-dotnet/pull/205

**Comment by jtattermusch** [comment]
Added a workaround that should make the interop tests pass.

**Comment by jtattermusch** [comment]
Known failures: https://github.com/grpc/grpc/issues/18637

**Comment by jtattermusch** [comment]
Merging to fix the breakage.

**Pull request merged**